packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -20,8 +20,8 @@ Name | Type | Default | Description
 -----|------|---------|------------
 cellComponent | ElementType&lt;[TableHeaderCellProps](#tableheadercellprops)&gt; | | A component that renders a header cell.
 rowComponent | ElementType&lt;[TableRowProps](table.md#tablerowprops)&gt; | | A component that renders a header row.
-allowSorting | boolean | false | Specifies whether a user can change the column's sorting state. Requires the [SortingState](sorting-state.md) dependency.
-showGroupingControls | boolean | false | Specifies whether to render controls that toggle the column's grouping state. Requires the [GroupingState](grouping-state.md) dependency.
+showSortingControls | boolean | false | Specifies whether to render controls that toggle the column's sorting state. Requires the [SortingState](sorting-state.md) dependency.
+showGroupingControls | boolean | false | Specifies whether to display a button that groups by the column. Requires the [GroupingState](grouping-state.md) dependency.
```

Comment:
Specifies whether to display a button that groups by the column. => Specifies whether to display a button that groups **rows** by the column.